SpeechResponder/eddi.zh-CN.json

```diff
@@ -1182,985 +1176,983 @@
     },
     "Nav beacon scan": {
-      "name": "Nav beacon scan",
-      "description": "Triggered when you scan a nav beacon, before the scan data for all the bodies in the system is written into the journal",
       "enabled": true,
       "priority": 3,
       "responder": true,
       "script": "{_ Context _}\r\n{SetState('eddi_context_last_subject', 'nav beacon')}\r\n{SetState('eddi_context_last_action', 'scan')}\r\n{SetState('eddi_context_star_system', system.name)}\r\n{SetState('eddi_context_nav_scans', event.numbodies)}\r\n\r\n{set nav to OneOf(\"导航\", \"航路\")}\r\n{set beacon to OneOf(\"信标\", \"球\")}\r\n{OneOf(\"这个 {nav} {beacon} 已经把本星系中 {event.numbodies} 个天体的详情传输给我了.\",\r\n \"我从这个{nav} {beacon}中收到了这个星系中 {event.numbodies}个天体的信息.\")}\r\n\r\n{Pause(500)}",
-      "default": true
+      "default": true,
+      "name": "Nav beacon scan",
+      "description": "Triggered when you scan a nav beacon, before the scan data for all the bodies in the system is written into the journal"
     },
     "Nav route": {
-      "name": "Nav route",
-      "description": "Triggered when the navigation route is updated",
       "enabled": true,
       "priority": 3,
       "responder": true,
       "script": "{_ Set to 0 to disable or set to 1 to enable optional parameters }\r\n{set reportDestination to 0}\r\n{set reportDistance to 0}\r\n{set reportJumps to 0}\r\n\r\n{_ Set optional details text from optional parameter values }\r\n{set details to []}\r\n{if reportJumps:\r\n   {set jumpDesc to: {event.jumps} {if event.jumps > 1: jumps |else: jump } }\r\n   {set details to cat(details, [jumpDesc])}\r\n}\r\n{if reportDistance:\r\n   {set distance to round(event.distance, 1)}\r\n   {if distance = 1:\r\n       {set distanceDesc to: {distance} 光年}   \r\n   |else:\r\n       {set distanceDesc to: {distance} 光年}\r\n   }\r\n   {set details to cat(details, [distanceDesc])}\r\n}\r\n\r\n{_ Begin speaking }\r\n{if event.jumps > 0:\r\n    {if reportDestination:\r\n        到 {P(event.route[event.jumps].systemname)}{if len(details) > 0:,}\r\n    }的航线已{OneOf(\"被绘制完成\", \"经设置好\", \"设定\", \"被标定\")},\r\n    {if len(details) > 0:\r\n        还有{List(details)} {OneOf(\"路程\", \"{if event.jumps > 1 || reportDistance: 剩余 |else: 剩余 }\")}\r\n    }.\r\n}",
-      "default": true
+      "default": true,
+      "name": "Nav route",
+      "description": "Triggered when the navigation route is updated"
     },
     "Near bookmark": {
-      "name": "Near bookmark",
-      "description": "Triggered when entering or departing the nearby radius of a bookmark",
       "enabled": true,
       "priority": 3,
       "responder": true,
       "script": "{if event.near:\r\n    已到达 \r\n|else:\r\n    正在离开\r\n}\r\n{if event.poi:\r\n    {P(event.poi, 'station')}.\r\n|else:\r\n    书签位置.\r\n}",
-      "default": true
-    },
+      "default": true,
+      "name": "Near bookmark",
+      "description": "Triggered when entering or departing the nearby radius of a bookmark"
+    },,
```

Comment:
`},,` may be an error. Should be `},`?

Reply:
If EDDI is unable to read this then the personality might be marked as malformed.